torch/_torch_docs.py

```diff
@@ -4489,6 +4489,42 @@ def merge_dicts(*dicts):
     ),
 )
 
+add_docstr(
+    torch.from_file,
+    """
+from_file(filename, shared=None, size=0, *, dtype=None, layout=torch.strided, device=None) -> Tensor
```

Comment:
This technically has `pin_memory` as an argument but I'm avoiding documenting it since `pin_memory` will always fail if set